xtask/src/release.rs

```diff
@@ -74,7 +74,8 @@ impl ReleaseArgs {
         //
         // More info: https://git-cliff.org/docs/usage/monorepos
         if self.cmd != ReleaseCommand::Changelog {
-            let _p = pushd(workspace::root_path()?)?;
+            let sh = sh();
+            let _p = sh.push_dir(workspace::root_path()?);
```

Comment:
I am not sure if I understand the initial intent here. Because `pushd` / `Shell::push_dir` returns a RAII guard, it stop having an effect as soon as it is out of scope. And it is out of scope right after being created.

Reply:
The intention was to not change the directory to the workspace if the `Changelog` command is used.

A `let _p;` before the if branch give the guard the correct lifetime. Could you add that please?

Reply:
Done.